src/app/leverage/components/faq-section/index.tsx

```diff
@@ -5,84 +5,235 @@ import { FaqItem, FaqList } from '@/components/faq'
+          through rebalancing within defined bounds. This automated rebalancing
+          enables liquidation protection and creates a hands-off experience for
+          users.
+        </p>
+        <p>
+          Leverage tokens can also be substantially cheaper than perps because
+          the borrow rates on Aave are significantly lower than the typical
+          funding rates on perp platforms.
+        </p>
+      </FaqItem>
+      <FaqItem question='Is there liquidation risk?'>
+        <p>
+          Every token in the Index Coop Leverage Suite automatically rebalances
+          the underlying collateralized debt positions on Aave v3 to avoid
+          liquidation during volatile periods. Redundant keeper systems monitor
+          the current leverage ratio for each token at all times and trigger a
+          rebalance if it exceeds the maximum leverage ratio.
+        </p>
+        <p>
+          Another layer of protection is enabled by the ripcord system, which
+          allows anyone to trigger a rebalance if the current leverage ratio
+          exceeds a predefined ripcord leverage ratio; there is also a 0.25 ETH
+          reward for anyone who triggers a ripcord rebalance.
+        </p>
+      </FaqItem>
+      <FaqItem question='How do I open a position?'>
+        <p>
+          Start by selecting a market via the dropdown in the quickstats widget
+          or in the trade widget, then select your desired Leverage amount using
+          the Leverage buttons. Now select your collateral from the ‘you pay’
+          dropdown menu. You will have to connect your wallet and sign our terms
+          and conditions before your first action. Make sure you are connected
+          to Arbitrum and that you have sufficient funds in your wallet. Now you
+          can input the desired amount of collateral and click ‘Review
+          Transaction’ which will open a transaction review popup. Click “Submit
+          Transaction”. This will run a transaction simulation and in case of a
+          successful simulation send it to your wallet. Use your wallet to sign
+          the transaction. If you want to see your new balance, click “Sell” in
+          the transaction window and your balance will show under the input
+          asset dropdown.
+        </p>
+      </FaqItem>
+      <FaqItem question='How do I close a position?'>
+        <p>
+          For leverage tokens on Arbitrum, you can close your position by first
+          selecting ETH or BTC, selecting “Sell” at the top of the trade widget,
+          and then selecting the corresponding leverage ratio of your currently
+          open position. You should see your total balance and be able to input
+          the amount of your position you want to close out.
+        </p>
+        <p>
+          For leverage tokens on Mainnet, you can sell via the Index Coop App{' '}
+          <Link href='/swap' className='underline' target='_blank'>
```

Comment:
Not sure if these `/swap` link shouldn't open in the same page or open a new tab?